CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [7.1.0](https://github.com/theforeman/puppet-qpid/tree/7.1.0) (2021-03-30)
```

Comment:
Sadly it's 2021-03-31 already :( I really wish there was some better way for it.